test/testpaint/compat.c

```diff
@@ -240,3 +240,8 @@ bool is_csg_loaded()
 {
     return false;
 }
+
+uint8 track_element_get_colour_scheme(const rct_map_element * mapElement)
+{
+    return mapElement->properties.track.colour & 0x3;
+}
```

Comment:
tsk tsk

Reply:
@janisozaur I am surprised you haven't make CIs fail builds over this yet.

Reply:
I don't understand what you two are on about. Isn't this the proper fix?

Reply:
@Gymnasiast it's missing newline. It doesn't show the beautiful icon unless you look at the commit page directly.

![screenshot-2017-10-4 clean up more track flags by gymnasiast pull request 6365 openrct2 openrct2](https://user-images.githubusercontent.com/550290/31168160-b677c79c-a8f4-11e7-8d48-df8824a74d51.png)

Reply:
Oh, that? I already fixed it for track.c, but overlooked this one.